cli/cmd/plugin/ui/web/tanzu-ui/src/views/unmanaged-cluster/unmanaged-cluster-wizard-page/unmanaged-cluster-settings-step/UnmanagedClustersSettingsAdvanced.tsx

```diff
@@ -40,24 +36,25 @@ const unmanagedClusterAdvancedSettingStepFormSchema = yup
     .required();
 
 interface FormInputs {
-    [UNMANAGED_ADVANCED_FIELDS.CLUSTER_NAME]: string;
-    [UNMANAGED_ADVANCED_FIELDS.CONTROL_PLANE_NODE_COUNT]: string;
-    [UNMANAGED_ADVANCED_FIELDS.WORKER_NODE_COUNT]: string;
+    [UNMANAGED_CLUSTER_FIELDS.CLUSTER_NAME]: string;
+    [UNMANAGED_CLUSTER_FIELDS.CONTROL_PLANE_NODE_COUNT]: string;
+    [UNMANAGED_CLUSTER_FIELDS.WORKER_NODE_COUNT]: string;
 }
 
 const unmanagedClusterProviders = [
     {
         label: 'kind',
```

Comment:
Could leverage the K8sProviders.constants.tsx values here

Reply:
<img width="398" alt="Screen Shot 2022-07-20 at 3 45 52 PM" src="https://user-images.githubusercontent.com/106271458/180068567-80fb0cf8-87c6-4f5c-8d59-e579074dbdde.png">

Is this what you meant by leverage the values?